lib/sift/client.rb

```diff
@@ -295,6 +295,114 @@ def score(user_id, opts = {})
     end
 
 
+    # Fetches the latest score(s) computed for the specified user and abuse types.
+    #
+    # As opposed to client.score() and client.rescore_user(), this *does not* compute
+    # a new score for the user; it simply fetches the latest score(s) which have computed.
+    # These scores may be arbitrarily old.
+    #
+    # See https://siftscience.com/developers/docs/ruby/score-api/get-score for more details.
+    #
+    # ==== Parameters:
+    #
+    # user_id::
+    #   A user's id. This id should be the same as the user_id used in
+    #   event calls.
+    #
+    # opts (optional)::
+    #   A Hash of optional parameters for the request --
+    #
+    #   :abuse_types::
+    #     List of abuse types, specifying for which abuse types a
+    #     score should be returned.  By default, a score is returned
+    #     for every abuse type to which you are subscribed.
+    #
+    #   :api_key::
+    #     Overrides the API key for this call.
+    #
+    #   :timeout::
+    #     Overrides the timeout (in seconds) for this call.
+    #
+    # ==== Returns:
+    #
+    # A Response object containing a status code, status message, and,
+    # if successful, the user's score(s).
+    #
+    def get_user_score(user_id, opts = {})
```

Comment:
@quanpod @garylee1 do you think we should raise an exception if version is < 205? This could apply to other methods in our other SDKs.